### PR TITLE
Remove 'archived' texts

### PR DESCRIPTION
**Pull request by Nicofuma**

### CODE REVIEW

**Comment by Nicofuma** [comment]
rebased

File: src/SensioLabs/JobBoardBundle/Resources/views/Includes/full_job_header.html.twig [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <a href="#">{{ job.city }}, {{ job.country|country }}</a> /
         <a href="#">{{ job.contractType|contractType }}</a>
 
-        <br/><span class="archived">Archived since 12/12/2013</span>
+        <!--<br/><span class="archived">Archived since 12/12/2013</span>-->
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Resources/views/Includes/full_job_header.html.twig]: why not using twig comments ?

File: src/SensioLabs/JobBoardBundle/Resources/views/Includes/shortened_job_header.html.twig [hunk 2]
```diff
@@ -3,7 +3,7 @@
         <a href="#">{{ job.city }}, {{ job.country|country }}</a> /
         <a href="#">{{ job.contractType|contractType }}</a>
 
-        <br/><span class="archived">Archived since 12/12/2013</span>
+        <!--<br/><span class="archived">Archived since 12/12/2013</span>-->
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Resources/views/Includes/shortened_job_header.html.twig]: same here

**Comment by Nicofuma** [comment]
updated

**Comment by salahm** [comment]
:+1:

**Pull request merged**